internal/cli/atlas/cloudproviders/accessroles/aws/create.go

```diff
@@ -29,9 +29,9 @@ import (
 
 const (
 	provider       = "AWS"
-	createTemplate = `AWS IAM role '{{.RoleID}}' successfully created.
-Atlas AWS Account ARN: {{.AtlasAWSAccountARN}}
-Unique External ID: {{.AtlasAssumedRoleExternalID}}
+	createTemplate = `AWS IAM role '{{.CloudProviderAccessAWSIAMRole.RoleId}}' successfully created.
```

Comment:
Will the CLI actually start displaying a different value, like `Rolename.RoleID`? If not (and this is just a new variable for the same RoleID), this change causes the automation we put in place for outputs to not work correctly. You would need to remove lines 68-70 of this file.

Reply:
@sarahsimpers it will be the same roleId just new variable name. I noticed that docs use unresolved templates in documentation (so this will be affected) but actual value never changes.

Reply:
Updated the template in the Annotations so the docs generated remain the same.
I did not create a separate variable just to avoid confusion with the output template.